kqueen_ui/gunicorn.py

```diff
@@ -1,8 +1,9 @@
-from kqueen_ui.config import current_config
+from kqueen_ui.config.utils import kqueen_ui_config
+
 
 import multiprocessing
 
-app_config = current_config()
+app_config = kqueen_ui_config
```

Comment:
this is useless, we can just use app_config everywhere. It is UI repo, so it is only one config should be here, no need to call it so complex as kqueen_ui_config